py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -832,6 +835,15 @@ def contains_metadata(gm: torch.fx.GraphModule) -> bool:
 
     dryrun_stats_display(dryrun_tracker, settings.dryrun)
 
+    if len(dryrun_tracker.to_run_in_torch) > 0:
```

Comment:
I enabled wrapper module when there is graph break.
@narendasan Do you think we need another runtime api to enable wrapped module?

Reply:
I think we could have a compile setting that is something like `run_with_cudagraphs` or something that returns the wrapped module. But I think its confusing that we have both, I think at some point we might want to just pick one